app/services/users.js

```diff
@@ -0,0 +1,29 @@
+const bcrypt = require('bcrypt');
+const errors = require('../errors');
+const logger = require('../logger');
+const { User } = require('../models');
+
+const createParams = params =>
+  bcrypt
+    .genSalt(10)
+    .then(salt =>
+      bcrypt.hash(params.password, salt).then(password => ({
+        firstName: params.first_name,
+        lastName: params.last_name,
+        mail: params.mail,
+        password
+      }))
+    )
+    .catch(error => logger.error(error));
```

Comment:
You not have a proper error handling here, because you're only logging the error, you should throw an internal error